.github/workflows/api-tests.yml

```diff
@@ -37,8 +37,11 @@ jobs:
             ${{ runner.os }}-build-
             ${{ runner.os }}-
 
+      - name: Node Version
+        run: node -v 
+
       - name: Install packages
-        run: npm install
+        run: npm ci 
```

Comment:
We need to commit the package-lock to use npm ci, otherwise I think it errors.

Reply:
can we just delete the package-lock?